scripts/automation/tests/run.py

```diff
@@ -40,5 +38,21 @@ def run_all():
 
     return passed
 
+
 if __name__ == '__main__':
-    sys.exit(0 if run_all() else 1)
+    parse = argparse.ArgumentParser('Test tools')
+    parse.add_argument('--module', dest='modules', action='append',
+                       help='The modules of which the test to be run. Accept short names, except '
+                            'azure-cli, azure-cli-core and azure-cli-nspkg')
+    parse.add_argument('--non-parallel', action='store_true',
```

Comment:
Good suggestion. That will be my next task.

Please remember that your scripts will keep working. Nothing is changed about the test cases here, therefore, they still can be run with python -m unittest.